### ISSUE TITLE
Landing page elements for design

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/open-design#1]
From @riksalazar:

> I believe these are the elements to add:
> - The logo
> - Navigation
> - Some description of what the landing is about with headline and description text.
> - List of open projects and open source projects
> - Some latest news from the Blog.
> - Footer

We'd also like to:
- showcase certain open projects
- Some description of the Ribose Open mission (this is the "landing about" you mentioned above)

Seems reasonable to me. Thoughts @ribose-jeffreylau @strogonoff ?

**Comment by strogonoff** [comment]
> showcase certain open projects
> Some description of the Ribose Open mission (this is the "landing about" you mentioned above)

Yeah, I believe that’s actually in your @ronaldtse original plan of pages to work on. I think that list was great and more than specific, I’ll post it here for the sake of reference for now.

```
== Site Structure
=== Landing Page
The purpose of the "Ribose Open" landing page is to:

highlight our commitment to open source and open specifications
provide a list of open projects / open source projects

** (or a partial list, e.g., featured + pagination of complete list)
link to our "Open blog" (or maybe visibility on the latest news?)

=== Open Blog

Each blog post is tagged to different project or project components
A blog post could be news on an old or new project, new or updated software or specifications

=== Open Project Index

List out all Open Projects

=== Open Project Individual Page

List out all open source software and open specifications that relate / belong to this Open Project.
blog that displays only blog posts about this project (i.e., link to Open Blog with a tag filter)

=== Open Source Software Index

List of open source projects
Each project with link to source code (e.g., GitHub)
A project contains one or more open source code projects

=== Open Source Software Individual Page

introduction
installation
usage
development documentation (API Docs)
link to source code, GitHub status like number of Stars, Pull Requests
people who have adopted this

=== Open Specifications Index

List of open specifications

=== Open Specifications Individual Page

specification page (just like the CSD standard, a document with multiple sections)
links to published documents like IETF drafts, or ISO documents
development tools and documentation relating to this specification
other people who have adopted this

```

**Comment by strogonoff** [comment]
The plan is to prioritize pages and attend to things in order, we want to focus on delivering an MVP early and then iterating on it (and enabling Ribose’s team to contribute to iteration).

First milestone would likely include the landing page, top-level blog, project’s landing page, and individual spec paper and software docs. (We might shuffle that around depending on situation, e.g. push project page for later and instead have spec and software listings earlier.)

It may be worth pointing out that _the blog_ would actually be the multi-purpose tool that can do a lot around publicity-related stuff, like showcasing and highlighting certain projects and achievements (even in absence of a dedicated feature on the landing page if that takes longer time to get right). Overall linking to latest blog articles from the landing page is a more fundamental feature, and search engines tend to weigh original blog articles, and links to such, higher than just generic list of highlights.
